lib/planner/write-plans-to-disk.js

```diff
@@ -7,11 +7,14 @@ const crypto = require('crypto');
 
 module.exports = function writePlansToDisk(plans) {
   const outputPath = configHandler.get('outputPath');
+  let index = 0;
   for (let plan of plans) {
     const sha256 = (x) => crypto.createHash('sha256').update(x, 'utf8').digest('hex');
-    let testName = `simulato`+`--${sha256(JSON.stringify(plan))}.json`;
+    let planString = JSON.stringify(Array.from(plan));
+    let testName = `simulato`+`--${index}--${sha256(planString)}.json`;
```

Comment:
I forget why this index was added? Just to keep track of the number?

Reply:
Yeah, specifically to make it easier to check for missing tests.

Reply:
Great